server/graphman/src/resolvers/deployment_mutation.rs

```diff
@@ -65,4 +70,45 @@ impl DeploymentMutation {
 
         restart::run_in_background(ctx, store, deployment, delay_seconds).await
     }
+
+    /// Unassign a deployment
+    pub async fn unassign(
+        &self,
+        ctx: &Context<'_>,
+        deployment: DeploymentSelector,
+    ) -> Result<Response> {
+        let ctx = GraphmanContext::new(ctx)?;
+        let deployment = deployment.try_into()?;
+
+        unassign::run(&ctx, &deployment)?;
+
+        Ok(Response::new(
+            true,
+            format!("Unassigned {}", deployment.as_str()),
+        ))
```

Comment:
We can replace this with `EmptyResponse` because the general expectation in APIs is that if a request is successful, it's done what it was supposed to do.